### PR TITLE
Added new bio for Vincent Cavanna

### PR DESCRIPTION
**Pull request by vcavanna**
bababoey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,7 +49,15 @@ <h2>Rosie the Kraken</h2>
...
 
       <!-- paste what you copied directly below here -->
-     
+     <div class="about">
+        <div class="bio-left">
+          <img src="rosie-the-kraken.png"/>
+        </div>
+        <div class="bio-right">
+          <h2>Vincent Cavanna</h2>
+          <p> I'm Vincent Cavanna! In the 1940s, I encouraged women to join the World War II effort in the United States. Since then, I have become a symbol of women participating in all sorts of industries where they are underrepresented. We can do it! </p>
```
Comment (Crushtoe) [inline index.html:58]: Fact check: Untrue.

**Review by Crushtoe — CHANGES_REQUESTED** [review]
you're no symbol

**Review by Sebas-2000 — APPROVED** [review]
You should add a better bio

File: index.html (hunk 1 above)
Comment (Sebas-2000) [reply index.html:58]: Agreed you've created some conflicts

**Pull request merged**